handbook/upgrade/upgrade.md

```diff
@@ -106,6 +106,15 @@ For single node upgrades, NI recommends upgrading and migrating at the same time
 
 1. Run the command `nislmigrate restore --all --secret <your secret> --dir D:\migration`.
 
+    !!!note
+        If migrating from SystemLink 21.3 or earlier to SystemLink 21.5 or later, your Test Monitor data must be migrated from MongoDB to PostgreSQL before the service can start. If Test Monitor is using the local instance of MongoDB stored in the default location, the migration will occur automatically. If not, the migration must be approved on the TestMonitor tab in the **NI SystemLink Server Configuration** application.
+
+        Depending on the size of your data set this process may take some time. For reference, a typical server takes less than one hour to migrate 5 million steps. To check the step count on your server, you can run the following command in Robo 3T: `db.getCollection('steps').stats()`. Use the step count to roughly estimate the expected migration time. Note that system resources and network connectivity will impact the migration time.
```

Comment:
I've interacted with customers that know their way around Robo 3T, but I wouldn't say the typical user is familiar. Should I provide a link to Robo 3T and mention the credentials can be found in TestMonitor.json? Or should I add a more involved guide here?

Reply:
I would defer to @prestwick.

Reply:
Any thoughts on this, @prestwick ?

Reply:
Good question. I would prefer this to be written a more generic manner than specifying Robo 3T. For example this could also be done from the Mongo shell.

Reply:
Also, this `!!!note` is getting a little long. 

- The intro to this note can likely be moved up to the intro before the workflow starts since this is information we'd want to equip the user with before they start the process. 
- The piece about observing the **migration** status can likely be moved to a step in the workflow.